x-pack/filebeat/processors/aws_vpcflow/testdata/custom-nat-gateway.golden.yml

```diff
@@ -0,0 +1,116 @@
+- aws:
```

Comment:
Curious why about why you're using YAML here.

Reply:
Probably because I had already gotten YAML out of the toolbox and didn't want to reach for the encoding/json. 😆 (this was a Friday night hackathon kind of idea). I'll switch the golden files over to JSON.